CMakeLists.txt

```diff
@@ -85,4 +86,4 @@ sofa_create_package(ModelOrderReduction ${PROJECT_VERSION} ${PROJECT_NAME} Model
 
 
 # Allows to use CPack (for the future)
-include(CPack)
\ No newline at end of file
+include(CPack)
```

Comment:
Finally you uncomment it ?

Reply:
Heh... no, It's just that I started from the initial CMakeLists.txt file, so I only re-applied the minimal changes.
Commented or not, this line doesn't do anything at this point, so I guess we can leave it like that.. :)